extensions/azurecore/src/azureDataGridProvider.ts

```diff
@@ -72,7 +75,8 @@ export class AzureDataGridProvider implements azdata.DataGridProvider {
 			{ id: 'type', type: 'text', field: 'typeDisplayName', name: loc.resourceType, width: 150 },
 			{ id: 'type', type: 'text', field: 'resourceGroup', name: loc.resourceGroup, width: 150 },
 			{ id: 'location', type: 'text', field: 'locationDisplayName', name: loc.location, width: 150 },
-			{ id: 'subscriptionId', type: 'text', field: 'subscriptionName', name: loc.subscription, width: 150 }
+			{ id: 'subscriptionId', type: 'text', field: 'subscriptionName', name: loc.subscription, width: 150 },
+			{ id: 'actions', type: 'actions', field: '', name: '' }
```

Comment:
maybe we don't need to ask user to define an actions column, there can be a table wide setting, e.g. showActionsColumn, and then you can get the contributed actions when the button is pressed.

Reply:
Yeah that's what I was thinking in my comment above. I was trying to avoid having a bunch of extra options like that but I think it's reasonable to have provider-wide options like that.

Reply:
I ended up just making the actions column always show. For the work I'm doing that'll always be the case - and then later on when/if we decide we need to support it not being shown we can do the work then to allow that.